### PR TITLE
Hotfix db tests

### PR DESCRIPTION
**Pull request by martinjonsson01**
## Summary
Makes the repository tests not connect to the same database that is used for the web application. 
Instead, it cleanses and creates a new test database each time the tests are run.

The workflow on GitHub is now also updated to install a PostgreSQL service before running our application, so that the database tests can run.

### CODE REVIEW

**Comment by martinjonsson01** [comment]
This is not a PR into master, but a PR into the branch for #83, so it isn't technically required to have 2 reviewers to be merged. 

I won't merge until I've got two accepted reviews, though.

File: src/Infrastructure/InjectionExtensions.cs [hunk 1]
```diff
@@ -26,20 +29,32 @@ public static class InjectionExtensions
...
+               });
+    }
+
+    private static void ConfigureDatabase(DbContextOptionsBuilder options)
+    {
+        // Note: this connection string should be stored in an environment variable away from the source code.
+        // If you are replacing this connection string with actual credentials to a real database, don't
+        // just replace the string here in the source code, use an environment variable instead.
+        _ = options.UseNpgsql(
+            "User ID=postgres;Password=password;Host=localhost;Port=5432;Database=contract_portal;");
```
Comment (jonathan-lindqvist) [inline src/Infrastructure/InjectionExtensions.cs:58]: yes

**Review by jonathan-lindqvist — APPROVED** [review]
yes

File: src/Infrastructure/InjectionExtensions.cs (hunk 1 above)
Comment (martinjonsson01) [reply src/Infrastructure/InjectionExtensions.cs:58]: yes

**Review by JacobBredin — APPROVED** [review]
hmm yes

**Pull request merged**